packages/mui-material-next/src/Select/Select.d.ts

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { SxProps } from '@mui/system';
+// TODO v6: replace @mui/material with @mui/material-next when components are available
+import { InternalStandardProps as StandardProps, Theme } from '@mui/material';
```

Comment:
`InternalStandardProps` is marked as [`"ONLY USE FROM WITHIN mui/material-ui"`](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/index.d.ts#L26) 🤔. Should we refactor this, or is it okay to import it from `material-next`?

Reply:
I think we could port this to `material-next` as we're still conforming to the same `describeConformance`, though I'm not really sure what the `TODO` really means 😓

Reply:
Improved the TODO comments 👍🏼 
I think it's ok to import `InternalStandardProps` from `material` for now